packages/playground/docs/landing-page.html

```diff
@@ -99,11 +99,11 @@ <h2 class="heading">Try out the demo apps</h2>
                         <div class="card-top">
                             <img class="framework-logo margin-bottom" src="{{ "/assets/images/react.svg" | absolute_url }}" alt="React Logo">
                             <p class="framework-description margin-bottom">We have a demo in React...</p>
-                            <a class="custom-link" target="_blank" href="https://github.com/SAP/ui5-webcomponents-sample-react">Explore
+                            <a class="custom-link" target="_blank" href="https://github.com/SAP-samples/ui5-webcomponents-sample-react">Explore
                                 the code</a>
                         </div>
                         <div class="card-bottom">
-                            <a class="custom-link bold-link" target="_blank" href="https://sap.github.io/ui5-webcomponents-sample-react/">RUN
+                            <a class="custom-link bold-link" target="_blank" href="https://sap-samples.github.io/ui5-webcomponents-sample-react//">RUN
```

Comment:
sometimes not slashed at the end, sometimes two, the links are working, up to you to unify them